core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -116,11 +116,10 @@ object ZkSecurityMigrator extends Logging {
     try {
       run(args)
     } catch {
-        case e: Exception => {
+        case e: Exception =>
           e.printStackTrace()
           // must exit with non-zero status so system tests will know we failed
           Exit.exit(1)
-        }
     }
```

Comment:
I'm not completely sure if it'll work as before after removing the curly brackets. Could you confirm it?

Reply:
From: https://docs.scala-lang.org/style/control-structures.html#curlybraces

> case - Always omit braces in case clauses.

I also double checked with one of the most used Scala code bases in concurrent world (Pekko, fork of Akka) that this is the case. You can see this in this class for example (one of the key classes of Pekko):
https://github.com/apache/incubator-pekko/blob/e597a702b9a54a0e193e09a50a1d45dfd1f56785/actor/src/main/scala/org/apache/pekko/actor/ActorCell.scala#L659

This is an instance where Java and Scala differ (in Java this diff would cause a different behaviour indeed).